CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+##Contributing
```

Comment:
Yessir. Thanks for compromising.. I think it will help people who contribute across multiple repos inside our orgs.